app/views/content/landing/social-mailing-list-sign-up/_header.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= render Content::CampaignHeroComponent.new(
+  title: ["Sign up for", "emails"],
+  colour: @front_matter["colour"],
```

Comment:
do we want a blue/pink colourway or do the social team specifically want pink/yellow?

if blue/pink, @garethwilliamsdesign has defined a colourway called `grow` which could be used here

Reply:
Done.